packages/web/components/Modal/CurateTechnologyModal/index.js

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { MdClose as CloseIcon } from 'react-icons/md';
+import useSWR from 'swr';
+import { useRouter } from 'next/router';
+
+import { TechnologyProvider } from '../../Technology/TechnologyProvider';
+import Loading from '../../Loading';
+import { TabList, TabPanel, Tabs as Container } from '../../Tab';
+import {
+	getAttachments,
+	getTechnologyCosts,
+	updateTechnologyCurationStatus,
+} from '../../../services/technology';
+import tabs from './tabs';
+import {
+	StyledModal,
+	TabsHeader,
+	StyledTab,
+	CloseButton,
+	ReviewWrapper,
+	ReviewTitle,
+	ReviewInput,
+	ReviewActions,
+	ReviewButton,
+} from './styles';
+import { normalizeTaxonomies } from '../../../utils/technology';
+import { toast } from '../../Toast';
+import { STATUS as statusEnum } from '../../../utils/enums/technology.enums';
+
+const CurateTechnologyModal = ({ closeModal, technology = {} }) => {
+	const [assessment, setAssessment] = useState('');
+	const [inputValue, setInputValue] = useState('');
+	const [isSubmitting, setIsSubmitting] = useState(false);
+	const router = useRouter();
+
+	const { data: technologyDetails = [], isValidating } = useSWR(
+		'getTechnologyDetails',
+		() =>
+			Promise.all([
+				getTechnologyCosts(technology.id, { normalize: true }),
+				getAttachments(technology.id, { normalize: true }),
+			]),
```

Comment:
Aqui na chave é bacana termos algo único pra cada situação. Creio que da forma como está agora o cache poderá ser aplicado incorretamente quando outra tecnologia for aberta pois a chave será a mesma.
```suggestion
		['getTechnologyDetails', technology.id],
		(_, id) =>
			Promise.all([
				getTechnologyCosts(id, { normalize: true }),
				getAttachments(id, { normalize: true }),
			]),
```